plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.test.tsx

```diff
@@ -73,11 +73,11 @@ describe('ScoreCard-TestWithData', () => {
   class MockClient implements ScoringDataApi {
     getScore(
       _entity?: Entity | undefined,
-    ): Promise<SystemScoreExtended | undefined> {
+    ): Promise<EntityScoreExtended | undefined> {
       throw new Error('Method not implemented.');
     }
-    getAllScores(): Promise<SystemScoreExtended[] | undefined> {
-      return new Promise<SystemScoreExtended[] | undefined>(
+    getAllScores(): Promise<EntityScoreExtended[] | undefined> {
```

Comment:
if we want to test also backwards compatible api's we shall mock also only SystemScoreExtended[] return...which brings me to the idea to throw away systemXxxx and use just entityXxxx... I will create a separate thread for this..